app/src/api/api.js

```diff
@@ -48,10 +48,20 @@ export const replyMessage = (message, messageConversationId) =>
         throw error;
       });
 
+export const deleteMessageConversation = messageConversationId =>
+  getD2Instance()
+    .then(instance =>
+      instance.Api.getApi().delete(`messageConversations/${messageConversationId}`))
+    .then(result => result)
```

Comment:
Why did you add all these lines?

Reply:
Changed from multi-delete. The api does not support multi-delete, but I will probably create some epic to handle multi-selection delete at some stage.